scripts/eurostat/gdp/import_data.py

```diff
@@ -142,10 +162,12 @@ def generate_tmcf(self):
                     continue
                 col_num += 1
                 if "HAB" in col:
-                    var = "dcid:Amount_EconomicActivity_GrossDomesticProduction"\
-                          "_AsAFractionOfCount_Person"
+                    var = ("dcid:Amount_EconomicActivity_"
+                           "GrossDomesticProduction_AsAFractionOfCount_"
```

Comment:
Amount_EconomicActivity_GrossDomesticProduction_Nominal_PerCapita

Reply:
Done.

Reply:
It looks like you changed GrossDomesticProduction_AsAFractionOfCount_Person_Nominal" to "GrossDomesticProduction_AsAFractionOfCount_Person" instead of what I put above. Was that intentional? Maybe some miscommunication? We can do a video sync if needed.